packages/gateway/src/metrics.js

```diff
@@ -184,11 +213,15 @@ export async function metricsGet(request, env, ctx) {
     ),
     `# HELP nftgateway_requests_per_time_total`,
     `# TYPE nftgateway_requests_per_time_total histogram for total of requests per response time bucket`,
-    ...histogram.map((t) => {
+    ...responseTimeHistogram.map((t) => {
       return env.ipfsGateways
         .map(
           (gw) =>
-            `nftgateway_requests_per_time_total{gateway="${gw}",le="${t}",env="${env.ENV}"} ${metricsCollected.ipfsGateways[gw].responseTimeHistogram[t]}`
+            `nftgateway_requests_per_time_total{gateway="${gw}",le="${msToS(
```

Comment:
also made the `le` with seconds as we use seconds everywhere for Prometheus best practises